testsuite/features/step_definitions/common_steps.rb

```diff
@@ -855,7 +855,7 @@
   node = get_target(client)
   if client.include? 'sle'
     node.run('zypper --non-interactive install wget', timeout: 500)
-  else # As Ubuntu has no support, must be CentOS/SLES_ES
+  else # As Debian-like has no support for trad clients, must be Red Hat-like
```

Comment:
```suggestion
  else # As Debian-like has no support for traditional clients, it must be Red Hat-like
```

Reply:
yup, we speak well too often in "telegraphic style"

Reply:
I'm very good with this style!

Reply:
LOL

Reply:
done